headlines/src/headlines.rs

```diff
@@ -24,13 +24,15 @@ pub enum Msg {
 pub struct HeadlinesConfig {
     pub dark_mode: bool,
     pub api_key: String,
+    pub refresh_news_data: bool,
```

Comment:
This need not be part of `HeadlinesConfig` as it's a state that is applicable while the app is running.